src/codegen/src/main/scala/WrapperClassDoc.scala

```diff
@@ -0,0 +1,277 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.codegen
+
+import org.apache.commons.lang3.StringUtils
+
+import Config._
+
+/** Provide class level python help documentation for generated classes.
+  * Lookup the doc string based on the name of the scala class
+  * The text for this help is drawn from the scaladocs explanations in the scala classes.
+  *
+  * Where possible, there is also sample code to illustrate usage.
+  *
+  * The default case, TODO + classname will help identify missing class docs as more modules are added
+  * to the codebase. When new classes are added, please add a case and docstring here.
+  */
+
+object WrapperClassDoc {
+  def GenerateWrapperClassDoc(className: String): String = {
+    className match {
+      case "AssembleFeatures" =>
```

Comment:
Can't we take these values from a file as opposed to encoding them in source? Each time we add a new class, it's very disruptive to have to change the wrapper generator.